tests/modules/nf-core/arcashla/extract/test.yml

```diff
@@ -0,0 +1,22 @@
+- name: arcashla extract test_arcashla_extract
+  command: nextflow run ./tests/modules/nf-core/arcashla/extract -entry test_arcashla_extract -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/arcashla/extract/nextflow.config
+  tags:
+    - arcashla
+    - arcashla/extract
+  files:
+    - path: output/arcashla/test.paired_end.sorted.extracted.1.fq.gz
```

Comment:
I would suggest adding a `contains` for a read header of a read you expect to find in there ofr all FASTQ files